Makefile

```diff
@@ -91,6 +91,22 @@ $(NODE_G_EXE): config.gypi out/Makefile
 	$(MAKE) -C out BUILDTYPE=Debug V=$(V)
 	if [ ! -r $@ -o ! -L $@ ]; then ln -fs out/Debug/$(NODE_EXE) $@; fi
 
+CODE_CACHE_DIR ?= out/$(BUILDTYPE)/obj.target
+CODE_CACHE_FILE ?= $(CODE_CACHE_DIR)/node_code_cache.cc
```

Comment:
I think most all generated code is in `out/$(BUILDTYPE)/obj/gen`?

Reply:
@addaleax Yes, not sure why I picked `obj.target`...thanks for pointing that out, I'll fix it